src/components/grid/NextMoveTile.tsx

```diff
@@ -0,0 +1,21 @@
+import { CORNER_POSITION, SURFACE_ELEVATION } from '../constants';
+import { Coordinates } from './SquareTypes';
+
+type PropsNextMove = {
+  coordinates: Coordinates;
+  handleMovePlayer: () => void;
+};

```

Comment:
I think you can name this component something like `BoardTile.tsx` and then use it in `PlayerTile.tsx`, because now there is duplicate logic